src/integrations.py

```diff
@@ -243,12 +256,12 @@ class IngressData:
     """The data source from the ingress integration."""
 
     is_ready: bool = False
-    url: str = DEFAULT_BASE_URL
+    url: str = f"http://{socket.getfqdn()}:{ADMIN_SERVICE_PORT}"
 
     def to_env_vars(self) -> EnvVars:
         return {
-            "BASE_URL": self.url,
-            "OAUTH2_REDIRECT_URI": urljoin(self.url, OAUTH_CALLBACK_PATH),
+            "CONTEXT_PATH": urlparse(self.url).path,
+            "OAUTH2_REDIRECT_URI": join(self.url, OAUTH_CALLBACK_PATH),
```

Comment:
Just a **NONBLOCKING** question: why do we change to the `os.path.join`?

Reply:
`urljoin` discards the path of the first argument (see https://stackoverflow.com/a/10893427)